.github/workflows/ci-aqua-security-trivy-tests.yml

```diff
@@ -1,6 +1,26 @@
 name: ci-aqua-security-trivy-tests
 on:
   push:
+    paths-ignore:
+      - '.vscode/CODEOWNERS'
+      - '.vscode/*.json'
+      - 'compose/**'
+      - 'contrib/**'
+      - 'dgraphtest/**'
+      - 'licenses/**'
+      - 'paper/**'
+      - 'RFC/**'
+      - 'static/**'
+      - 't/**'
+      - 'telemetry/**'
+      - 'wiki/**'
+      - '**.dockerignore'
+      - '**.gitignore'
+      - '**/**.md'
+      - '**/**.png'
+      - '**/**.sh'
```

Comment:
this is overly broad and should be removed

Reply:
This what? When you don't point out exactly what you mean. I can't tell if it is about .sh or all lines from 16 to 21.
If it is about .sh, yeah. I put it there to discuss. As I see we are more and more going away from .sh. And all .sh there is there are deprecated or just examples.

Reply:
Sorry if that wasn't clear; I was referring to the last line `**/**.sh`.  There are still a few places where we are using shell scripts (e.g. protobuf generation).  Also in general it is overly broad not to run CI against shell scripts.